### PR TITLE
Instruction for installing pyenv

### PR DESCRIPTION
**Pull request by pan3793**
`pyenv` is a quite hygienic and convenient solution to manage multiple isolated Python virtualenvs, this PR updates the README to instruct users to setup pyenv on macOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,41 @@ of the Hadoop ecosystem to facilitate the rapid establishment of test environmen
 
 # Prepare
 
-This project uses [ansible](https://www.ansible.com/) to build. Please make sure you have installed it before building.
+This project uses [ansible](https://www.ansible.com/) to to render the Dockerfile, shell scripts, and configuration files from the templates. Please make sure you have installed it before building.
```
Comment (yanghua) [inline README.md:8]: double `to`

File: README.md [hunk 2]
```diff
@@ -5,9 +5,41 @@ of the Hadoop ecosystem to facilitate the rapid establishment of test environmen
...
-Considering, ansible strongly depends on the Python environment. To make the Python environment independent and easy to manage, it is recommended to use `pyenv` and `virtualenv` to manage Python environment.
+Considering, ansible strongly depends on the Python environment. To make the Python environment independent and easy to manage, it is recommended to use `pyenv` and `virtualenv` to manage Python environment on macOS.
+
+Install from Homebrew
+
+```bash
+brew install pyenv pyenv-virtualenv
+```
+
+Append to `~/.zshrc`
```
Comment (yanghua) [inline README.md:18]: It seems not everyone uses `zsh` or knows what is `.zshrc`. Give some description or suggestion or remove?

File: README.md [hunk 3]
```diff
@@ -5,9 +5,41 @@ of the Hadoop ecosystem to facilitate the rapid establishment of test environmen
...
+
+```bash
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+
```
Comment (yanghua) [inline README.md:24]: If we change `~/.zshrc`, add another step to `source` it?

File: README.md (hunk 2 above)
Comment (yanghua) [reply README.md:18]: Reference this [install guide](https://gist.github.com/josemarimanio/9e0c177c90dee97808bad163587e80f8#installation), shall we append pyenv to the path environment variable?

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+cffi==1.15.0
+cryptography==37.0.2
+Jinja2==3.1.2
+lxml==4.9.1
+MarkupSafe==2.1.1
+packaging==21.3
+pycparser==2.21
+pyparsing==3.0.9
+PyYAML==6.0
+resolvelib==0.5.5
```
Comment (yanghua) [inline requirements.txt:12]: It seems we do not need all of these packages?

**Review by yanghua — COMMENTED** [review]
Left some comments.

File: README.md (hunk 2 above)
Comment (pan3793) [reply README.md:18]: `zsh` is the macOS default shell

Comment (pan3793) [reply README.md:18]: brew handles the PATH stuff

File: requirements.txt (hunk 4 above)
Comment (pan3793) [reply requirements.txt:12]: I dumped it using pip

File: README.md (hunk 3 above)
Comment (pan3793) [reply README.md:24]: updated

File: README.md (hunk 1 above)
Comment (pan3793) [reply README.md]: updated

**Comment by pan3793** [comment]
Merged first, and I can open the follow-up to address further comments

**Pull request merged**